### ISSUE TITLE
Possible Issue with Parameter Passing in generate_feature.py

### ISSUE THREAD

**Issue by Wang-cb** [EdisonLeeeee/SpikeNet#9]
Hello, author.

I have a question regarding the parameter passing in generate_feature.py:

model = DeepWalk(80, 10, 128, window_size=10, negative=1, workers=16)

Compared to the __init__ function in deepwalk.py:

def __init__(self, dimensions: int = 64,
             walk_length: int = 80,
             walk_number: int = 10,
             workers: int = 3,
             window_size: int = 5,
             epochs: int = 1,
             learning_rate: float = 0.025,
             negative: int = 1,
             name: str = None,
             seed: int = None):

It seems that 128 should represent the feature dimensions, but it is actually being passed to walk_number. Could this be a mistake?

Looking forward to your response. Thank you!
